### ISSUE TITLE
Enhance our sync for external users

### ISSUE THREAD

**Issue by nadjaweisskopf** [leihs/leihs#706]
We discussed the following problem: 

- external employes get an email address (though HRM)
- they do not get a ldap-login
- our leihs-sync excludes these external employees

After a discussion with @eins78 the best solutio would be the following: 
- our Sync includes external employees (with email address)
- if an external employee needs to access leihs, HRM needs to send a request for an ldap-login
- we sync the employee with the email, login and ID
- user can log into leihs

Therefor as soon as the users login does not work anymore after he has left ZHdK, his leihs login does not work anymore either.

We have a ticket from a leihs user to this matter. @DrTom Could you please inform Franziska as soon as you have adjusted the sync? https://service.itz.zhdk.ch/otrs/index.pl?Action=AgentTicketZoom;TicketID=255252

**Comment by eins78** [comment]
@drtom We want to add the affected user manually, for now. Ideally they would use the AGW login. Challenge: Can we add the user with the needed properties so that AGW login works BUT the sync will not deactivate this account on next run (because it won't be in the list from ZAPI). If not possible, we will add a password manually.

**Comment by DrTom** [comment]
Let me rephrase first:

There are users in the (evento) database which are not passed to the leihs sync because of some business rules. 

We want still want to enable them so they can use the AGW (or Switch AAI).

Yes this is possible: set the proper `email` and `login` values! Do not set the `org_id` value!

There is a catch: whenever one of those users will be changed so it will be passed on via the ZAPI API the sync will fail (and it would be complex and dangerous to fix that). I would suspect that it is not uncommon that these kind of users will be enabled. Or ist it not?

**Comment by nadjaweisskopf** [comment]
Until these certain external users can not be flagged seperately, the sync can not be adjusted. Otherwise we import far too many users which do not have a relevance for leihs.

**Comment by DrTom** [comment]
With the switch from `zhdk_only` to `has_account` everybody with a sign-in should be able to access leihs.
